### PR TITLE
Change termination nomenclature

### PR DESCRIPTION
**Pull request by ghost**
Changed Quit to Exit on the menu.

### CODE REVIEW

File: arcom.py [hunk 1]
```diff
@@ -102,7 +102,7 @@ def print_menu(status):       ## Your menu design here
     print " ", 8 * "-", "%6.6s - Arcom RC210 Control" % identity, 8 * "-"
     printStatus(status)
     print " |", 44 * " ", "|"
-    print " | 0.  Quit                                     |"
+    print " | 0.  Exit                                     |"
```
Comment (dpkingston) [inline arcom.py:105]: This great.

Comment (ghost) [reply arcom.py:105]: Fantastic.

Comment (dpkingston) [reply arcom.py:105]: OK.  We're done.  LGTM.

**Review by dpkingston — APPROVED** [review]
LGTM.

**Pull request merged**